src/solc_0.8/bundleSandSale/PolygonBundleSandSale.sol

```diff
@@ -292,4 +281,17 @@ contract PolygonBundleSandSale is WithAdmin, IERC1155TokenReceiver {
         uint256 saleId = sales.length;
         emit BundleSale(saleId, ids, amounts, sandAmountPerPack, priceUSDPerPack, numPacks);
     }
+
+    function _getSaleAmount(uint256 saleId, uint256 numPacks)
+        internal
+        returns (uint256 saleIndex, uint256 usdRequired)
+    {
+        require(saleId > 0, "invalid saleId");
```

Comment:
I like the OZ like revert message ie "PolygonBundleSandSale: invalid saleId" it's easier when debugging

Reply:
Clearly this looks better and is easier to debug, but we have not been using that format anywhere else in the code and
I wanted the code to look consistent so I went ahead with the simple message. Should we create a ticket and do this throughout the code base of solc_0.8?

Reply:
I already started to write it like this. I'm not a big fan of changing all at once. I prefer starting using the new rule for the new code.

Reply:
Okay, I'll update the comment then